html/themes/custom/common_design_subtheme/templates/layout/page.html.twig

```diff
@@ -1,3 +1,4 @@
+{% set attribuutes = attributes.addClass('cd-content--spacious') %}
```

Comment:
two 'u's

Reply:
Thanks for catching this typo. Interestingly, that doesn't prevent the class from being added.